test/e2e/mcd_test.go

```diff
@@ -546,3 +546,91 @@ func TestDontDeleteRPMFiles(t *testing.T) {
 		}
 	}
 }
+
+func TestFIPS(t *testing.T) {
```

Comment:
non-blocking suggestion: it might be nice to have this test structured as a table test with a couple variations of machineconfig sets.

Reply:
yeah, that would be amazing but test timeout is 75m today and going through various reboots for applying the MC + other tests would exceed that